### ISSUE TITLE
Add Version of the VST3 sdk

### ISSUE THREAD

**Issue by ycollet** [COx2/glslEditor_AudioPlugin#6]
Can you add the version of the VST3 sdk ?
I tried several and it looks like the good one is around 3.7.4 ... But still struggling with the sdk version

**Comment by COx2** [comment]
Hi, thanks for your comment.
Is this a request for replacing a newer version of the VST3 SDK?
I thought it would be better in general to update the version of JUCE itself if I were to deal with that issue.

If I can find the time, I will consider upgrading the version of JUCE itself. If I do that, I might also consider improving the project structure in general.

**Comment by ycollet** [comment]
Yes, I will vote for a JUCE update :) Maybe using cmake with juce will ease the build of the program too ...
It looks like the compilation of the plugin is really sensitive to the patch version of the vst3 sdk.
